src/target/riscv/riscv-011_reg.c

```diff
@@ -0,0 +1,60 @@
+#ifdef HAVE_CONFIG_H
```

Comment:
[Just a note, no change requested]

It is possible that it would make sense to put more register-related functions into the `riscv-011_reg` and `riscv-013_reg` modules. However, I do not yet have a clear opinion on this topic.

So let's leave it as is in this PR. The first step for the code division has been made and it can be further refined as we go.